### ISSUE TITLE
[BUG] Grid settings possibly being corrupted, leading to UI display issues

### ISSUE THREAD

**Issue by danrazor** [RupertAvery/DiffusionToolkit#191]
hi,
I mentioned this before with examples but there is a difference that i have found.
in your new three PANE (column?) 
[Folder tree mode>explorer mode> image preview section]
if an image has metadata, then it is pushed outside the area of the prompt in the preview section. 
image is pushed to the far right so much that it is outside the visible window and if we try to zoom in or out then only the left side of the image seem to appear but only extremely zoomed in.
but if an image does not have any metadata then it will be viewed as usual in the preview area.

this is a real problem.
kindly check if this is the same case on your system.
cheers

**Comment by RupertAvery** [comment]
Can you post an example image here?

**Comment by danrazor** [comment]
![Uploading image_2024-01-24_114047277.png…]()

**Comment by danrazor** [comment]
but images with METADATA or PNGchunks or pnginfo (which are the most important images for diffusion toolkit after all)
they are not visible in the preview area.
if we zoom in and out then some zoomed in part of the image can be seen.
it seems the bottom information section is forcing the top image preview area to push image out of the frame.

![image_2024-01-24_114631284](https://github.com/RupertAvery/DiffusionToolkit/assets/41794256/c265d1f9-aa01-4394-98ce-9418a13160bb)

**Comment by danrazor** [comment]
Funny thing is that if you hide the PROMPT in the information section then the image is visible.
![image_2024-01-24_134610162](https://github.com/RupertAvery/DiffusionToolkit/assets/41794256/1d153ff4-652b-4d0f-84b8-b5d0a11ceeba)

**Comment by RupertAvery** [comment]
Can you post the image file (JPG/PNG) that causes this? Not the screenshot. I'd like to check if it is something related to data, or isolated to your setup.

**Comment by RupertAvery** [comment]
As you can see, this is how it appears on my screen:

![image](https://github.com/RupertAvery/DiffusionToolkit/assets/1910659/3f6ab9a4-1a7d-4f5e-985c-7126a3baaf62)

![288650695-8ff58e5f-320c-4d51-85ae-1b1f40b33079](https://github.com/RupertAvery/DiffusionToolkit/assets/1910659/6689b481-7d3c-49d3-8cd6-8dfd307ed990)

**Comment by RupertAvery** [comment]
Alternatively, you can try going to  Settings > Database > Open Database Folder

Then, important, close DIffusion Toolkit first.

Then open config.json and delete the following:

```
  "MainGridWidth": 
  "MainGridWidth2": 
  "NavigationThumbnailGridWidth": 
  "NavigationThumbnailGridWidth2": 
  "PreviewGridHeight": 
  "PreviewGridHeight2":
```

Restart DT.

This should reset the layout.  Some users seem to have been affected by something similar.

**Comment by danrazor** [comment]
yes that solved the issue, i think

**Comment by RupertAvery** [comment]
This should be fixed in 1.6
